src/script.sh

```
@@ -198,24 +209,39 @@ main() {
     fi
 
     ## Setup 
     ### puts inputs in /home/dnanexus/in/
     dx-download-all-inputs
 
+    ### Here is the best place to do the index checking,
+    ### because the next step makes all of the DNANexus
+    ### variables I'm using here obsolete
+    if [[ "$run_CollectVariantCallingMetrics" == "true" ]]; then
+        if [[ $vcf_name == *.vcf ]]; then
+            bgzip $vcf_path
```

Shellcheck (0.11.0)

[warning] vcf_path is referenced but not assigned.

(SC2154)